### DISCUSSION TITLE
Hot reloading, live reload extension

### DISCUSSION THREAD

**Discussion by fsz4** [casid/jte#392] [Ideas]
Hi, it was my first time trying JTE today and it is amazing, but mostly when using other web frameworks there is hot reloading which also automatically reloads the page in the browser.
From what I see in the docs, there is no such thing currently in JTE.

It would be cool to have a live reload option in the future :)

**Comment by edward3h** [comment]
Welcome!

JTE is really only for rendering templates. It is agnostic to which web framework it is used with, or even whether it is used for web pages.

A framework with live reload needs more integration with the web server, and javascript injected into the page, so that updates can be pushed. I hope that makes sense.

**Comment by wimdeblauwe** [comment]
I just released ttcli 1.9.0 which allows to create a Spring Boot project with JTE setup that does live reloading (based on Vite). See https://github.com/wimdeblauwe/ttcli/releases/tag/1.9.0

It would be great if you could give it a test.
